### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by djalova**
Removed `node_modules` folder from repo
Added gitignore
Changed a path in `controller.js` to use relative path instead of absolute. There is an issue where the app tries to load from an invalid path and crashes the model.

### CODE REVIEW

File: public/js/controller.js [hunk 1]
```diff
@@ -57,9 +57,9 @@ function start() {
 
     var mycamvas = new camvas(window.ctx, processFrame);
     initialized = true;
-    stats.showPanel(0);
-    stats.dom.style.left = '10px';
-    stats.dom.style.top = '200px';
+    // stats.showPanel(0);
```
Comment (ajbozarth) [inline public/js/controller.js]: why comment and not remove?

**Review by ajbozarth — APPROVED** [review]
LGTM

**Review by autoih — APPROVED** [review]
LGTM

File: public/js/controller.js (hunk 1 above)
Comment (djalova) [reply public/js/controller.js]: I thought maybe for debugging purposes but I can just remove it to be cleaner.

**Pull request merged**